src/components/Calculator/Digit/Digit.tsx

```diff
@@ -0,0 +1,44 @@
+import './digit.css';
+
+const Digit = ({ appendDigit }: { appendDigit: (n: DigitNumbers) => void }) => {
+  return (
+    <div className="digits flex">
+      {DIGIT_NUMBERS.map((number) => (
+        <button
+          className="digit"
+          key={number}
+          onClick={() => appendDigit(number)}
+        >
+          {number}
+        </button>
+      ))}
+    </div>
+  );
+};
+
+export default Digit;
+
+export type DigitNumbers =
+  | '9'
+  | '8'
+  | '7'
+  | '6'
+  | '5'
+  | '4'
+  | '3'
+  | '2'
+  | '1'
+  | '0';
+
+const DIGIT_NUMBERS: ReadonlyArray<DigitNumbers> = [
+  '9',
+  '8',
+  '7',
+  '6',
+  '5',
+  '4',
+  '3',
+  '2',
+  '1',
+  '0',
+];
```

Comment:
이렇게 작성하면 똑같은 값을 타입과 상수에 두 번 선언하는 게 되어버리네요.

타입 > 상수가 아니라 상수 > 타입은 어떨까요?

```suggestion
const DIGIT_NUMBERS = ['9', '8', '7', '6', '5', '4', '3', '2', '1', '0'] as const;
type DigitNumbers = typeof DIGIT_NUMBERS[number];
```